### ISSUE TITLE
Discrete Motion Issue

### ISSUE THREAD

**Issue by CumulusAlpha** [JoSangYeon0709/mycobot_320_gripper_moveit#1]
Hello, I tried using the package you developed, but when I used demo.launch to control the real robotic arm, I found that the arm moved intermittently, showing a kind of discrete motion. Do you have any suggestions for improving this?

**Comment by JoSangYeon0709** [comment]
Hello.
Which demo.launch in the package did you run?
This repository only changed the necessary parts to operate the urdf file and gripper from the code supported by Elephant Robot.


If you want to use ros1 noetic and move mycobot 320 through moveit,
Terminal 1
roslaunch mycobot_320_gripper_moveit mycobot_320_gripper_moveit.launch
Terminal 2
rosrun mycobot_320_gripper_moveit sync_plan.py
Please do so. If you still have problems, please ask again.

And do you have any video or photos of the irregular movements?

**Comment by CumulusAlpha** [comment]
Sorry, i mistook the _mycobot_320_gripper_moveit mycobot_320_gripper_moveit.launch_ as _demo.launch_ ,which is created by Moveit_Setupassistant.
Let me describe my problem again:
The real scene is that i did the steps as you say in `readme.md`, I `roslaunch mycobot_320_gripper_moveit` in Teminal1 and `rosrun mycobot_320_gripper_moveit sync_plan.py` in Terminal2.
When I dragged the blue ball in RViz to control the manipulator, the robot started to move, but it did not move smoothly but ... how to say... maybe like it moves and stops and then moves and stops....it looped like this until it finally went to destination. 
So, how did you make it to control the robot to move smoothly? I'd appreciate it if you can help me!

**Comment by JoSangYeon0709** [comment]
When I moved the position of the end effector in rviz and did plan & execute,

Are you talking about the gray mycobot 320 moving while being cut off?

I think that problem can be solved by changing the settings in MotionPlanning -> Plannend Path to make the animation smooth.

If the robot doesn't follow properly when you move the blue ball to change the position of the end effector, it's because the end effector is in a position and direction that it can't go. Most problems are caused by positions and directions that the end effector can't go to.

But is it not the situation I mentioned above, and does the actual robot move strangely?
In that case, I need to know what the environment is so that I can reproduce it. That way I can help you.

Oh, and I fixed the robot's base stl file because it was in a strange shape.
I only changed the base.stl file, so if you download the file again, the robot base will appear properly in rviz.
